tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/ContentPublishingServiceTests.Publish.cs

```diff
@@ -630,6 +630,21 @@ public async Task Cannot_Publish_Non_Existing_Culture(string cultureCode)
         Assert.AreEqual(ContentPublishingOperationStatus.InvalidCulture, result.Status);
     }
 
+    [Test]
+    public async Task Can_Publish_Invariant_Content_With_Cultures_Provided_If_The_Default_Culture_Is_Included()
```

Comment:
Can_Publish_Invariant_Content_With_Cultures_Provided_If_The_Default_Culture_Is_Included
=>
Can_Publish_Invariant_Content_With_Cultures_Provided_If_The_Default_Culture_Is_Exclusively_Provided

Should also add one where it is actually included, as in, multiple valid cultures with one of them being the default culture.

Same for unpublish